skcosmo/__init__.py

```diff
@@ -1 +1,4 @@
+from .datasets import load_degenerate_manifold
+
+__all__ = ["load_degenerate_manifold"]
```

Comment:
I would not import `load_degenerate_manifold` in the main `__init__.py`, since this is not something users would use, but more something for tests & examples. 

If it is not imported here, then users have to do `from skcosmo.dataset import ...`, which should be better overall

Reply:
removed